src/store/debug/debug-reducers.ts

```diff
@@ -0,0 +1,13 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { pushAction } from './debug-actions';
+import { debugInitialState, DebugState } from './debug-state';
+
+const MAX_RECENT_ACTIONS = 20;
+
+export const debugReducers = createReducer<DebugState>(debugInitialState, builder => {
+  builder.addCase(pushAction, (state, { payload }) => ({
+    ...state,
+    recentActions: [...state.recentActions.slice(-MAX_RECENT_ACTIONS + 1), payload]
```

Comment:
`ActionArrivalPayload` and `pushAction` names still does not match with `recentActions` & `MAX_RECENT_ACTIONS`

Reply:
Could you offer your own variants?

Reply:
https://github.com/madfish-solutions/templewallet-mobile/pull/181#discussion_r684003591